app/components/Notifications/Notifications.styled.ts

```diff
@@ -0,0 +1,42 @@
+import { BG_SECONDARY_SPECIAL_4, BORDER_PRIMARY_LIGHT, BORDER_SECONDARY_LIGHT, TEXT_SECONDARY } from 'style-constants';
+
+const styles = {
+  containerStyles: {
+    position: 'absolute',
+    padding: '0 18px',
+    fontSize: '16px',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    borderBottom: `1px solid ${BORDER_SECONDARY_LIGHT}`,
+    alignItems: 'start',
+    justifyContent: 'center',
```

Comment:
you can use className="pa fz16 df fdc aic jcc" in Notification.jsx => same for all styled files